src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -605,6 +778,46 @@ def testLargeJob(self):
         self.assertIsNot(job1, job2)
         self.assertEqual(job2.command, command)
 
+    @classmethod
+    def _getUrlForTestFile(cls, size=None):
+        from toil.jobStores.azureJobStore import _fetchAzureAccountKey
+        fileName = 'testfile_%s' % uuid.uuid4()
+        containerName = cls._externalStore()
+        url = 'wasb://%s@%s.blob.core.windows.net/%s' % (containerName, cls.accountName, fileName)
+        if size is None:
+            return url
+        blobService = BlobService(account_key=_fetchAzureAccountKey(cls.accountName),
+                                  account_name=cls.accountName)
+        content = os.urandom(size)
+        blobService.put_block_blob_from_text(containerName, fileName, content)
+        return url, hashlib.md5(content).hexdigest()
+
+    def _hashJobStoreFileID(self, jobStoreFileID):
+        with self.master.readFileStream(jobStoreFileID) as f:
+            return hashlib.md5(f.read()).hexdigest()
+
+    @staticmethod
+    def _hashUrl(url):
+        from toil.jobStores.azureJobStore import AzureJobStore
+        blobService, containerName, blobName = AzureJobStore._extractBlobInfoFromUrl(urlparse.urlparse(url))
+        content = str(blobService.get_blob_to_bytes(containerName, blobName))
```

Comment:
Is `str` used to convert from unicode? If so, this should be explained in a short comment.